### PR TITLE
feat: [LI-52] implement ui for mobile confirmation page

### PR DESCRIPTION
**Pull request by lulu0729**
### Issue ticket number and link
[LI-52](https://predictivehire.atlassian.net/browse/LI-52)
### Figma Screenshot
image

### Describe main changes
- implement ui for mobile confirmation page
image

### Checklist in Local 
- [x] I have extract component business logic into custom hook
- [x] I have added unit tests
- [ ] I have integrated with backend api and done a e2e test locally
- [ ] I have updated the graphql query types

### Checklist in Dev
- [ ] I have deployed to dev
- [ ] I have integrated with backend api and done a e2e test

### Checklist before requesting a review
- [ ] I have done the **code format** using `pnpm lint:fix` and `pnpm format:fix`
- [ ] I have **rebase squash** current branch
- [ ] I have confirmed **Naming Convention** with team if needed.
- [ ] I have done a **self review** in GitHub.

### Checklist in Qa Environment
- [ ] I have deployed to qa and let QA team do a test
- [ ] I have fix the QA bugs after QA team finish testing
- [ ] I have put the screenshot for Qa environment integration test below:

### Checklist in Sandbox Environment
- [ ] I have deployed to sandbox and let QA team do test
- [ ] I have fix the Sandbox Bugs after QA team finish testing
- [ ] I have put the screenshot for Qa environment integration test below:

### Checklist after Merge
- [ ] I have known the dependencies of release deployment of current PR.
- [ ] I have reviewed the release pipe from team lead to secure the production deployment.
- [ ] I have update the Jira tickets status after all task has been finished.

[LI-52]: https://predictivehire.atlassian.net/browse/LI-52?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by yingzhi-lu** [comment]
Haha isn't the font weight a bit larger than Figma?

**Review by martin-luo — COMMENTED** [review]
I have some comments, please have a look

**Comment by lulu0729** [comment]
> Haha isn't the font weight a bit larger than Figma?

It was a bit much, so now I've reduced all the font weights 🤔

File: src/pages/schedule-interview/components/MobileCandidateConfirmation/MobileCandidateConfirmation.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+export const MobileCandidateConfirmation = () => {
+  const { interviewData } = useCandidateConfirmation()
+  const { jobRequisitionName, interviewMode, interviewAddress, timezone, interviewTime, interviewLink } = interviewData
+  const interviewDate = formatDateToLongString(new Date(interviewTime.start), timezone)
+  const interviewTimeSlot = formatDateToTimeSlot(interviewTime, timezone)
+
+  return (
+    <article
+      className={cn([
+        "candidateConfirmationWrapper h-screen overflow-y-auto bg-gray-100 p-5",
```
Comment (martin-luo) [inline src/pages/schedule-interview/components/MobileCandidateConfirmation/MobileCandidateConfirmation.tsx:21]: why do we still have `candidateConfirmationWrapper` here?
```suggestion
        "h-screen overflow-y-auto bg-gray-100 p-5",
```

File: src/pages/schedule-interview/components/MobileCandidateConfirmation/MobileCandidateConfirmation.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            <div className="mb-3 flex items-center">
+              <div className="text-tertiary w-[100px] text-sm font-light">Link</div>
+              <div className="ml-2 text-sm font-semibold text-black">{interviewLink}</div>
+            </div>
+          )}
+
+          <div className="flex items-center">
+            <div className="text-tertiary w-[100px] text-sm font-light">Timezone</div>
+            <div className="ml-2 text-sm font-semibold text-black">{timezone}</div>
+          </div>
```
Comment (martin-luo) [inline src/pages/schedule-interview/components/MobileCandidateConfirmation/MobileCandidateConfirmation.tsx:56]: all these 4 rows share the same DOM structure, we need to improve the code maintainability by either defining a function, or a component

File: src/pages/schedule-interview/components/MobileCandidateConfirmation/MobileCandidateConfirmation.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        </div>
+        <button className="flex h-[49px] w-full items-center justify-center rounded-full bg-black">
+          <CalendarOutlined className="calendarOutlined mr-2 text-2xl" />
```
Comment (martin-luo) [inline src/pages/schedule-interview/components/MobileCandidateConfirmation/MobileCandidateConfirmation.tsx:59]: ~~why we need `calendarOutlined` here?~~

If this is only for defining the `color: white`, I think we can use classname `!text-white` instead

**Review by bennyli519 — APPROVED** [review]
LGTM

File: src/pages/schedule-interview/components/MobileCandidateConfirmation/MobileCandidateConfirmation.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import { MobileCandidateConfirmationInfo } from "./MobileCandidateConfirmationInfo"
+
+export const MobileCandidateConfirmation = () => {
+  const { interviewData } = useCandidateConfirmation()
+  const { jobRequisitionName, interviewMode, interviewAddress, timezone, interviewTime, interviewLink } = interviewData
+  const interviewDate = formatDateToLongString(new Date(interviewTime.start), timezone)
+  const interviewTimeSlot = formatDateToTimeSlot(interviewTime, timezone)
+
+  return (
+    <article className="candidateConfirmationWrapper h-screen overflow-y-auto bg-gray-100 p-5">
```
Comment (martin-luo) [inline src/pages/schedule-interview/components/MobileCandidateConfirmation/MobileCandidateConfirmation.tsx:18]: why we still need `candidateConfirmationWrapper` here?

File: src/pages/schedule-interview/components/MobileCandidateConfirmation/MobileCandidateConfirmation.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+      <div className="h-auto w-full max-w-sm rounded-[20px] bg-white p-5 shadow-card">
+        <div className="mb-6 text-center">
+          <PHIcon size={140} name="interviewConfirmation" />
+          <h2 className="mb-2 mt-4 text-lg font-semibold text-black">
+            We look forward to seeing you on {`${interviewDate} ${interviewTimeSlot}`}!
+          </h2>
```
Comment (martin-luo) [inline src/pages/schedule-interview/components/MobileCandidateConfirmation/MobileCandidateConfirmation.tsx:24]: Can we do something like this? 
```suggestion
          <h2 className="mb-2 mt-4 text-lg font-semibold text-black">
            We look forward to seeing you on <time>{`${interviewDate} ${interviewTimeSlot}`}</time>!
          </h2>
```
The suggestion I made here is not complete, you need to provide the time value to it https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

**Review by martin-luo — APPROVED** [review]
Looks good to me, good job!

**Pull request merged**